arm-sql/2014-04-01/swagger/resourceDefinitions.json

```diff
@@ -0,0 +1,271 @@
+{  
```

Comment:
you could include these definitions in one of your existing files and have others refer to them, having a separate file will, at this time, produce "many" violations, since this swagger file is not following an RP structure. At some point, our tools would account for this separate files.